src/utils/executeELM.js

```diff
@@ -454,12 +454,13 @@ function updateSearchParams(params, release, type) {
       switch (type) {
         case "Questionnaire":
           params.set("_id", INSTRUMENT_LIST.join(","));
+          params.set("_count", 300);
```

Comment:
per meeting, this should be for QuestionnaireResponse, not Questionnaire

Reply:
thank you for catching that Justin. I did just fix it.